pkg/util/github/github.go

```diff
@@ -235,6 +248,28 @@ func DownloadOpenShiftReleaseBinary(binaryType OpenShiftBinaryType, osType minis
 	return nil
 }
 
+func listDirExcluding(dir string, excludeRegexp string) ([]string, error) {
```

Comment:
@hferentschik @praveenkumar , I mean as per Go, if you have same datatype of parameters in the function(like for `dir` and `excludeRegexp` have `string` as datatype) then you keep the datatype of leftmost parameter and omit rest as : 
```
func listdirExcluding(dir, excludeRegexp string) ([]string, error) {
....
}
```

Its a common idiom in Go.
Sorry I didn't elaborate my comment previously.
```